postgres/server/init.sls

```diff
@@ -15,7 +15,11 @@ include:
 
 {%- endif %}
 
-{%- set pkgs = [postgres.pkg] + postgres.pkgs_extra %}
+{%- if postgres.extensions %}
+  {%- set pkgs = [postgres.pkg] + [postgres.pkg_contrib] + postgres.pkgs_extra %}
```

Comment:
This statement looks very verbose and unnecessary...
It declares strait dependency between `extensions` setting in Pillar and newly introduced `pkg_contrib` value, which in turn seems to be applicable only on SUSE distros.

* What if desired extensions are not shipped with package defined in `pkg_contrib`?
* I see it was set empty (evaluates to `None`) for other platforms. Will it work if some extensions have been set?

It looks little bit confusing, and is not described anywhere.
I suggest to keep the things as is: if user needs some extension, he or she should provide extra packages via `pkgs_extra` or install them another way.

Reply:
Thanks @vutny The extensions have dependency on *some* package in PG and its gray area causing issues like #172 - suggestions welcomed there.  I agree to remove this from PR code.